azure-pipelines.yml

```diff
@@ -10,14 +10,14 @@ schedules:
 trigger:
 - master
 
-pool: '.Net Bubble - GCP'
+pool: '.Net Bubble staging'
```

Comment:
Now it worked most probably because there are not other images in the staging pool, however I think the proper way to target the image is to use also  demands: and say the PR number.